channel.go

```diff
@@ -1,14 +1,33 @@
 package thingfulx
 
-import (
-	"time"
-)
+import "time"
 
 // Channel represents an individual data channel being published by a Thing out
 // there in the world somewhere.
 type Channel struct {
-	ID         string    `json:"id"`
-	Value      string    `json:"value"`
+	ID           string            `json:"id"`
+	Metadata     []ChannelMetadata `json:"metadata"`
+	Observations []Observation     `json:"observations"`
+	Data         Data              `json:"data"`
+}
+
+// ChannelMetadata is an object which contains any metadata required to
+// semantically descibe a measurement collected by a Thing.
+type ChannelMetadata struct {
+	Prop string `json:"property"`
+	Val  string `json:"value"`
+}
```

Comment:
I imagine the use of the ChannelMetadata struct 

```{
 	Prop: "m3-lite:foo",
 	Val:  "bar",
 }```

would it be useful to maybe separate out the namespace e.g.

```
type ChannelMetadata struct {
        Schema string `json:"schema"` 
	Key string `json:"key"`
 	Value  string `json:"value"`
 }
```
so it looks like 

```{
 	Schema: "m3-lite",
        Key : "foo",
 	Value:  "bar",
 }```

This is a question - if you don't feel the same just ignore.